sdk/src/Core/Amazon.Runtime/_bcl/ClientConfig.cs

```diff
@@ -133,6 +133,34 @@ public WebProxy GetWebProxy()
             return proxy;
         }
 
+        /// <summary>
+        /// Returns a WebProxy instance to use for HTTPS connections if an
+        /// explicit web proxy hasn't been configured.
+        /// </summary>
+        public WebProxy GetHttpsProxy()
+        {
+            var httpsProxy = Environment.GetEnvironmentVariable("https_proxy");
```

Comment:
Was the proxy used during your test configured to use credentials? For example, `http://1:1@127.0.0.1:8888`
During my tests it didn't look like the credentials were automatically parsed from the environment variable when using the .NET Framework.

The SDK still supports `net35` and `net45`, so this is something we should fix.

Reply:
Hey! I did use credentials, but this was awhile ago, and I don't think I tested on many dotNet versions. Do you know where the credentials are getting dropped perchance? I can try and take a look at parsing credentials there, and associating them.

Reply:
Got it thanks! Let me try with that exact setup, and see what's going on, and make a change & test for it. Appreciate the callout.

Reply:
Hey! So this did indeed slip past, I think what happened is I originally tested my proxy very early on in debug mode, which was allowing no credentials through for debugging purposes at the time.

I've gone ahead and updated it to actually parse out the credentials, and it does now seem to pass them on successfully to the proxy! So marking this as resolved.